src/main/resources/ca/corefacility/bioinformatics/irida/export/ncbi.xml

```diff
@@ -17,10 +17,22 @@
 	</Description>
 
 	<Action th:each="sample : ${submission.bioSampleFiles}">
+		<!-- IRIDA sequence files may either be plaintext (.fastq) or gzip compressed (.fastq.gz)
```

Comment:
Just doing some final testing with this.  Can you make these "parser level" comments?  It will make them not appear in the final rendered xml.  It just means changing the comment block format a to `<!--/*  your comment here */-->`

See https://www.thymeleaf.org/doc/tutorials/2.1/usingthymeleaf.html#thymeleaf-parser-level-comment-blocks for more